packages/nodes-base/nodes/Airtop/actions/file/getMany.operation.ts

```diff
@@ -0,0 +1,95 @@
+import {
+	type IExecuteFunctions,
+	type INodeExecutionData,
+	type INodeProperties,
+} from 'n8n-workflow';
+
+import { requestAllFiles } from './helpers';
+import { wrapData } from '../../../../utils/utilities';
+import { apiRequest } from '../../transport';
+import type { IAirtopResponse } from '../../transport/types';
+
+const displayOptions = {
+	show: {
+		resource: ['file'],
+		operation: ['getMany'],
+	},
+};
+
+export const description: INodeProperties[] = [
+	{
+		displayName: 'Return All',
+		name: 'returnAll',
+		type: 'boolean',
+		default: false,
+		description: 'Whether to return all results or only up to a given limit',
+		displayOptions,
+	},
+	{
+		displayName: 'Limit',
+		name: 'limit',
+		type: 'number',
+		displayOptions: {
+			show: {
+				resource: ['file'],
+				operation: ['getMany'],
+				returnAll: [false],
+			},
+		},
+		typeOptions: {
+			minValue: 1,
+			maxValue: 100,
+		},
+		default: 10,
+		description: 'Max number of results to return',
```

Comment:
From the UI the user can't tell there is a min and max, it just gets autocorrected. Let's add it in the description. We could also say: `If you need to return more, consider using the 'Return All` setting`.
```suggestion
		description: 'Max number of results to return. Should be between 1 and 100.',
```

Reply:
There's another lint rule which requests exactly that description 😅
<img width="1110" alt="image" src="https://github.com/user-attachments/assets/3e80ebed-d951-4bc1-8b16-22b2c357078b" />

Reply:
This is how it behaves when entering invalid numbers, this might be sufficient for users to know the limits of valid numbers
![Screen Recording 2025-05-02 at 1 24 51 p m](https://github.com/user-attachments/assets/6935db9b-1dda-476e-a989-602b06d26802)